HoloLensCommander/HoloLensCommander/Controls/DeviceMonitorControlViewModelProperties.cs

```diff
@@ -36,7 +36,7 @@ private set
         /// Gets the battery level of this device, as a percentage remaining.
         /// </summary>
         /// <remarks>The returned string is formatted to two significant decimal places.</remarks>
-        private string batteryLevel = "";
```

Comment:
https://stackoverflow.com/questions/263191/in-c-should-i-use-string-empty-or-string-empty-or-to-intitialize-a-string